### PR TITLE
Link to EDM RDF serializer git hook instructions

### PR DESCRIPTION
**Pull request by pwin**

### CODE REVIEW

File: doc/gistStyleGuide.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Serialization
 -----
 
 - gist OWL files are serialized in RDF/XML. 
-- The serialization tool, `serialize.sh` (Mac/Linus) or `serialize.bat` (Windows), should be run before each commit in order to standardize formatting and eliminate noise in git diffs. It is recommended to automate this as a pre-commit hook.
+- The serialization tool, `serialize.sh` (Mac/Linus) or `serialize.bat` (Windows), should be run before each commit in order to standardize formatting and eliminate noise in git diffs. It is recommended to automate this as a pre-commit hook. Details on this are available from the [EDM Council](https://wiki.edmcouncil.org/display/FIBORDFKIT/Installing+and+Using+the+EDMC+RDF+Serializer)
```
Comment (rjyounes) [inline doc/gistStyleGuide.md:16]: Add final period.

Comment (rivettp) [reply doc/gistStyleGuide.md:16]: It's Linux not Linus (cept in Peanuts)

Comment (rjyounes) [reply doc/gistStyleGuide.md:16]: This was resolved in a separate PR #221.

**Comment by rjyounes** [comment]
Made some additions.

**Comment by uscholdm** [comment]
There is a pre-commit file in this folder: ...\Offerings\gist\git4gist\ which, I think, was a reworked one from edmc changed by Ted. Have a look.

**Comment by rjyounes** [comment]
So we should put this in the repo if this is the one we want to use.

**Comment by uscholdm** [comment]
Sure, but someone needs to verify that it is what we want. What are others using - if anything?  I thought I had this set up for myself, but apparently not.

**Pull request merged**